test/plugins/tokens.test.js

```diff
@@ -436,4 +440,19 @@ describe('token plugin test', () => {
             });
         });
     });
+
+    describe('Logging suppresses API tokens', () => {
+        it('does not print API tokens in logs', () => {
+            const source = new PassThrough({ objectMode: true });
+            const result = new PassThrough({ objectMode: true });
+
+            source.write(`This is a string with a token in it! ${testTokenWithValue.value}`);
+
+            source.pipe(suppressAPITokens).pipe(result);
+
+            result.on('data', (chunk) => {
```

Comment:
Typically streams are asynchronous, and this looks like an event listener. What are you doing to ensure that the test function isn't completing before the assertion happens? With mocha, this is typically accomplished by having a `done` callback in the test definition, which is called in the test when it should actually be completed.